modules/ztp-challenges-of-far-edge-deployments.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: CONCEPT
+[id="ztp-challenges-of-far-edge-deployments_{context}"]
+= Overcoming the challenges of deploying clusters at scale
+
+Today, service providers want to deploy distributed mobile networks as defined by the link:https://www.o-ran.org/specifications[O-RAN (Open Radio Access Network) specification]. The O-RAN specification provides a modular functional framework for 5G networks. Using O-RAN as a foundation, service providers can move from appliance-based radio access networks to open cloud based networks, gaining flexibility and agility in how they deliver services to end users.
```

Comment:
O-RAN is _one_ specification for interfaces between CNFs but any Open RAN specification allows for a disaggregated topology. This is however irrelevant for this document and only a cause for confusion

I suggest this section should be changed to describe a general geographically distributed workload such as e.g. a Cloud RAN

Reply:
I'll expand on this using the slide deck you provided David.

Reply:
I've adjusted this slightly to make it more generic and focused on the _OpenShift_ side of things as much as possible rather than the specific cloud RAN use case. http://file.emea.redhat.com/aireilly/td-522/scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-clusters-at-scale.html